### DISCUSSION TITLE
新建存储的时候,有个启用sign 的选项,为什么默认是不开启的?

### DISCUSSION THREAD

**Discussion by sparkssssssssss** [AlistGo/alist#5297] [Q&A]
起因添加存储后都是正常的,但是过一段时间后发现很多存储获取失败了,失败的都是没有开启sign的,我开启后就都正常了,请教一下这个作用是什么,开启了有什么影响?
谢谢!

**Comment by anwen-anyi** [comment]
有 设置的全部签名 和 元信息加密码 都会携带 `sign` 参数

**Reply by sparkssssssss** [reply]
那对于用户来说,有什么影响呢?
